### DISCUSSION TITLE
auto-tweeting feedback & ideas

### DISCUSSION THREAD

**Discussion by attritionorg** [restincode/restincode#174] [Ideas]
"I would also like to have the twitter account I created @RestInCode to tweet out via Lamda or whatever a reminder of dates of passing or other messages of well wishes on a regular basis."

I'd recommend pinging Steve Werby, who runs the @TodayInInfosec account which does just that. Not sure if he has it automated or if so, to what degree, but he can likely give valuable input on his experience here.

https://twitter.com/todayininfosec
https://twitter.com/stevewerby

**Comment by mubix** [comment]
Thanks! Reached out.

**Comment by devsecfranklin** [comment]
I can build some stuff for you guys if you want. :)

**Comment by mauvehed** [comment]
Twitter sucks now. Guess we need a Mastodon account now? infosec.exchange or defcon.social?

@mubix @attritionorg

**Comment by mubix** [comment]
Ya, I'm not sure the best place for this would be. Maybe Mastodon and possibly Twitter?

**Comment by attritionorg** [comment]
Twitter sucks but still highly used. Mastadon yes, but make sure a federated account. If Threads catches on and has an open API, there too at some point.
